index/index.js

```diff
@@ -24,85 +24,177 @@ function weekDiff(start, end) {
 }
 
 // Calls the banner proxy and returns class info in JSON format
-const invokeLambda = async(params) => {
-  const data = await lambda.invoke(params).promise();
-  data.Payload = data.Payload.replace(/\+/g, '');
-  return JSON.parse(data.Payload);
+async function invokeLambda(params) {
+  try {
+    const data = await cache.lambda.invoke(params).promise();
+    data.Payload = data.Payload.replace(/\+/g, '');
+    return JSON.parse(data.Payload);
+  } catch (err) {
+      console.log(err);
+      return err;
+  }
 }
 
-exports.handler = async(event, context) => {
-  const db = new AWS.DynamoDB.DocumentClient({ region: "us-east-2" });
-  
+// Calls the banner proxy and returns campues names & codes
+async function invokeCampusLambda(params) {
+  try {
+    const data = await cache.campusLambda.invoke(params).promise();
+    data.Payload = data.Payload.replace(/\+/g, '');
+    return JSON.parse(data.Payload);
+  } catch (err) {
+      console.log(err);
+      return err;
+  }
+}
+
+// Puts items into database 
+async function putIntoDB(item) {
+  try {
+    const data = await cache.db.put(item).promise();
+    console.log(data);
+   } catch (err) {
+    console.log(err);
+   }
+}
```

Comment:
BatchWrite would be a better choice than individually looping since you can do 25 items at once.